[2024-04-23T12:41:59.593+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates manual__2024-04-23T11:54:48.966898+00:00 [queued]>
[2024-04-23T12:41:59.599+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline_eq.gen_dates manual__2024-04-23T11:54:48.966898+00:00 [queued]>
[2024-04-23T12:41:59.599+0000] {taskinstance.py:2193} INFO - Starting attempt 4 of 4
[2024-04-23T12:41:59.610+0000] {taskinstance.py:2217} INFO - Executing <Task(_PythonDecoratedOperator): gen_dates> on 2024-04-23 11:54:48.966898+00:00
[2024-04-23T12:41:59.613+0000] {standard_task_runner.py:60} INFO - Started process 150015 to run task
[2024-04-23T12:41:59.615+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'pipeline_eq', 'gen_dates', 'manual__2024-04-23T11:54:48.966898+00:00', '--job-id', '290', '--raw', '--subdir', 'DAGS_FOLDER/kbs/custom_dags/pipeline_eq.py', '--cfg-path', '/tmp/tmp91vntufc']
[2024-04-23T12:41:59.616+0000] {standard_task_runner.py:88} INFO - Job 290: Subtask gen_dates
[2024-04-23T12:41:59.641+0000] {task_command.py:423} INFO - Running <TaskInstance: pipeline_eq.gen_dates manual__2024-04-23T11:54:48.966898+00:00 [running]> on host datalab-yasmine
[2024-04-23T12:41:59.669+0000] {logging_mixin.py:188} WARNING - /home/yasmine/.local/share/virtualenvs/knowledge-base-siaap-WvBZnYs-/lib/python3.8/site-packages/airflow/utils/context.py:207 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2024-04-23T12:41:59.807+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='pipeline_eq' AIRFLOW_CTX_TASK_ID='gen_dates' AIRFLOW_CTX_EXECUTION_DATE='2024-04-23T11:54:48.966898+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-23T11:54:48.966898+00:00'
[2024-04-23T12:41:59.808+0000] {rw.py:84} INFO - Starting to save files into Minio
[2024-04-23T12:42:03.184+0000] {python.py:202} INFO - Done. Returned value was: dates_20240423
[2024-04-23T12:42:03.194+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=pipeline_eq, task_id=gen_dates, execution_date=20240423T115448, start_date=20240423T124159, end_date=20240423T124203
[2024-04-23T12:42:03.241+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-23T12:42:03.256+0000] {taskinstance.py:3312} INFO - 1 downstream tasks scheduled from follow-on schedule check
